esphome/core/time.cpp

```diff
@@ -1,4 +1,6 @@
+#if defined(USE_DATETIME_DATE) || defined(USE_DATETIME_TIME)
```

Comment:
This can be simplified, otherwise I will need to add USE_DATETIME_DATETIME later when that entity is added too
```suggestion
#ifdef USE_DATETIME
```

Reply:
Okay, I was not sure about that.